java/ql/src/change-notes/2022-02-14-os-guards.md

```diff
@@ -0,0 +1,8 @@
+---
+category: minorAnalysis
+---
+ * Add new guards `IsWindowsGuard`, `IsSpecificWindowsVariant`, `IsUnixGuard`, and `IsSpecificUnixVariant` to detect OS specific guards.
+ * Add new predicate `getSystemProperty` that gets all expressions that retrieve system properties from a variety of sources (eg. alternative JDK API's, Google Guava, Apache Commons, Apache IO, ect..).
```

Comment:
```suggestion
 * Add new predicate `getSystemProperty` that gets all expressions that retrieve system properties from a variety of sources (eg. alternative JDK API's, Google Guava, Apache Commons, Apache IO, etc..).
```

Reply:
Maybe also the following?
- "eg." -> "e.g."
- "etc.." -> "etc."